typings/index.d.ts

```diff
@@ -1,95 +1,95 @@
-// Type definitions for CodeceptJS 1.0.2
+// Type definitions for CodeceptJS
 // Project: https://github.com/codeception/codeceptjs/
-// Definitions by: Michael Bodnarchuk <http://github.com/DavertMik>, Drew Diamantoukos <https://github.com/KennyRules>
+/// <reference path="./types.d.ts" />
 
-declare class Locator {
-  or(locator): Locator;
-  find(locator): Locator;
-  withChild(locator): Locator;
-  find(locator): Locator;
-  at(position): Locator;
-  first(): Locator;
-  last(): Locator;
-  inside(locator): Locator;
-  before(locator): Locator;
-  after(locator): Locator;
-  withText(text): Locator;
-  withAttr(attr): Locator;
-  as(output): Locator;
-}
+import * as Protractor from "protractor";
```

Comment:
What happens if i dont have protractor in node_modules?

Reply:
It should ignore it and give all defined const the 'any' type. Will double-check this behavior.

Reply:
I got error:
![image](https://user-images.githubusercontent.com/11293201/62383694-03ae9280-b559-11e9-99fc-0d6d7fec0cd4.png)